### ISSUE TITLE
Plotting flat maps + plotting additional data as transparency

### ISSUE THREAD

**Issue by jsmentch** [nilearn/nilearn#2658]
I am plotting Cifti files with [hcp-utils](https://rmldj.github.io/hcp-utils/) and nilearn. It works very nicely for the standard inflated surface but the aspect ratio is off for plotting a flatmap (squished to a square):

![download (42)](https://user-images.githubusercontent.com/20958589/104226981-ab9ae180-5416-11eb-9e32-1eed4c643a69.png)

![download (43)](https://user-images.githubusercontent.com/20958589/104226914-89a15f00-5416-11eb-86b2-12ff852f4b8f.png)

I'm so far unable to plot the bilateral brain flat map without this stretching happening. Is there interest in supporting plotting these flatmaps, maybe adding a keyword arg? And/or does anyone have tips on how to preserve the correct aspect ratio? I have been trying to modify the axes and fig.subplots_adjust but no luck.

Another related feature request is to be able to plot multiple stats on one surface map eg one type of data by color and another by transparency/alpha. On the above images for example, I've plotted my surf_data as normal and then blended it into the bg_map based on another map of r2score values. I could help to add this kind of functionality if it would be helpful and not complicate things too much. Thanks!

**Comment by bthirion** [comment]
Thx for the report !
What version of Nilearn are you using ? The 0.7.0 ?
Other wise, it would be great if you could share the flat map for us to reproduce and fix.

**Comment by jsmentch** [comment]
Sure! Yes this is v0.7.0 and looks like their surfaces are [here](https://github.com/rmldj/hcp-utils/tree/master/hcp_utils/data)

here is an example if hcp-utils is installed:

```
import hcp_utils as hcp
from nilearn import plotting
plotting.plot_surf(hcp.mesh.flat_right)
```

plot is not too stretched I think plotting one hemisphere
![download (44)](https://user-images.githubusercontent.com/20958589/104248726-0bee4b00-5438-11eb-9ec3-096cea4d6397.png)

plotting bilateral is squished:
`plotting.plot_surf(hcp.mesh.flat)
`
![download (45)](https://user-images.githubusercontent.com/20958589/104249181-e3b31c00-5438-11eb-9afe-4654367e1449.png)

**Comment by bthirion** [comment]
This is weird indeed. thank you for reporting.
Best,

**Comment by jeromedockes** [comment]
thanks! interestingly I can also reproduce it with `view_surf`, which uses completely independent machinery than `plot_surf`

**Comment by bthirion** [comment]
Could we have a short script to work on it ?

**Comment by jeromedockes** [comment]
> Could we have a short script to work on it ?

basically the script posted above by @jsmentch:
```python
import numpy as np
from matplotlib import pyplot as plt

import hcp_utils as hcp
from nilearn import plotting, surface

surf = surface.load_surf_mesh(hcp.mesh.flat)

plotting.plot_surf(surf)
plt.gcf().savefig("/tmp/plot-surf-hcp.png", bbox_inches="tight")
plotting.view_surf(surf).open_in_browser()
plt.close("all")


plt.scatter(surf.coordinates[::10, 1], surf.coordinates[::10, 2])
plt.gca().set_aspect(1)
plt.gcf().savefig("/tmp/scatter-nodes.png", bbox_inches="tight")
```

you will need to `pip install hcp-utils`

**Comment by bthirion** [comment]
Indeed, thx. @NicolasGensollen I think this is one for you, whenever you can. Best,
